### ISSUE TITLE
VB -> C#: Exception when converting a file

### ISSUE THREAD

**Issue by luedi** [icsharpcode/CodeConverter#553]
### Steps to reproduce
1. Select a VB source class in Solution explorer
2.  Open popup menu in Solution Explorer and execute "Convert to C#" command

### Error message shown
In Phase 2 the following exception is shown:
![image](https://user-images.githubusercontent.com/12098314/77994663-b0305680-732a-11ea-8893-a0fdb06781bf.png)

### Details
I'm using Visual Studio 2019 Community Edition with version 8.0.3 of the converter. The error occurs both on the latest production and preview release. 

I have debugged the addin and found the cause of the error on line 70 in ProjectConversion.cs. 

``` cs
 var codeResult = conversionResults.SingleOrDefault(x => !string.IsNullOrWhiteSpace(x.ConvertedCode)) ?? conversionResults.First();
```
`conversionResults` contains four items, the VB file and three resx files (see below). Because of this the `.SingleOrDefault` failed.

![image](https://user-images.githubusercontent.com/12098314/77994292-051f9d00-732a-11ea-938c-5e43b9d9aa8c.png)

I have attached a small solution which reproduces the error:

[CodeConverterError.zip](https://github.com/icsharpcode/CodeConverter/files/4407357/CodeConverterError.zip)

**Comment by GrahamTheCoder** [comment]
Thanks for the full bug report! 
This is a regression in the latest version, so you can install a previous version from releases, or to work around it you can probably click on the project and convert the whole thing (then just grab the files you want after and get rid of the rest of the changes). I'll get this fixed asap

**Comment by GrahamTheCoder** [comment]
I believe this is fixed in 95c60e6 Please let me know if you still see an issue with the latest master / CI build. Thanks!

**Comment by luedi** [comment]
This was a quite fast fix 😃 . I can confirm that the bug is fixed.
